### PR TITLE
first draft navbar, tried next Link object

### PR DESCRIPTION
**Pull request by christiangryt**

### CODE REVIEW

File: src/app/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            to: "./#",
+        },
+        {
+            title: "Statistics",
+            to: "./#"
+        },
+    ];
+
+    return (
+        <>
```
Comment (ShaileshS1702) [inline src/app/components/Header.tsx]: you can remove these as you only have one div

File: src/app/pizzadata.json [hunk 2]
```diff
@@ -50,4 +50,4 @@
         "price": 239,
         "img":"Peppersvennen.jpg"
     } 
-]
```
Comment (ShaileshS1702) [inline src/app/pizzadata.json]: Important!

Are you sure this one should be removed?

**Review by ShaileshS1702 — DISMISSED** [review]
Good code, check out review marked with Important! before merging

File: src/app/pizzadata.json (hunk 2 above)
Comment (christiangryt) [reply src/app/pizzadata.json]: Didnt intentionally change pizzadata. Will revert.

**Review by christiangryt — COMMENTED** [review]
LGTM!

**Pull request merged**